src/chainparams.cpp

```diff
@@ -377,11 +383,13 @@ class CRegTestParams : public CChainParams {
         consensus.signet_challenge.clear();
         consensus.nSubsidyHalvingInterval = 150;
         consensus.BIP16Exception = uint256();
-        consensus.BIP34Height = 500; // BIP34 activated on regtest (Used in functional tests)
+        // Dogecoin: Raise this a long way so it's below 3 * COINBASE_MATURITY still
+        consensus.BIP34Height = 1000; // BIP34 activated on regtest (Used in functional tests)
         consensus.BIP34Hash = uint256();
         consensus.BIP65Height = 1351; // BIP65 activated on regtest (Used in functional tests)
         consensus.BIP66Height = 1251; // BIP66 activated on regtest (Used in functional tests)
-        consensus.CSVHeight = 432; // CSV activated on regtest (Used in rpc activation tests)
+        // Dogecoin: Note this needs to be a multiple of nMinerConfirmationWindow
+        consensus.CSVHeight = 720; // CSV activated on regtest (Used in rpc activation tests)
```

Comment:
So....

Has to be a multiple of 144 unless we change the miner window for regtest (which means we're mining more blocks, with potential impact on Travis runtimes/timeout risk). 576 doesn't work because it's too low, with the increased block maturity, without massively editing a LOT of tests. 720 was next viable number.

Reply:
duplicate with below. closing this.